plutus-core/plutus-core.cabal

```diff
@@ -446,8 +446,10 @@ executable uplc
         Parsers
     build-depends:
         plutus-core -any,
+        aeson -any,
         base <5,
         bytestring -any,
+        containers -any,
```

Comment:
```suggestion
```

I wondered why we needed that, but in fact it's making the build fail.  I guess it'll need `updateMaterialized` too

Reply:
Wait, I can just update the nix file directly.  Let me try that.